### ISSUE TITLE
Bestiary Map

### ISSUE THREAD

**Issue by Cfourr** [runelite/runelite#13432]
**Is your feature request related to a problem? Please describe.**
Ever get tired of having to look up to find out where monsters are located that you've never heard of? Or maybe want to find out if there are better places to train than the one you're at? I certainly do, so I came up with this idea.

**Describe the solution you'd like**
A map similar to the in game map, but instead have each killable NPC listed to the left with clickable light up indicators on where they spawn.

**Describe alternatives you've considered**
I know that idea might be a lot, so I was also thinking about maybe instead a bestiary plugin that goes on the side bar where you can type in the name of the monster and it will pull up a list of the locations and maybe helpful info about that location (e.g: requirements to access area/kill, LVL of NPC, as well as maybe NPC's located nearby) (and maybe eventually integrate it to where you can click on the location name and it will pull up an instanced map of the location)
 
**Additional context**
I know this isn't exactly context about the project, but I would be more than willing to help out with any information gathering that would need to be done; if anyone is interested in this idea. I don't know anything about coding but I am good at data collection and organization. =)

**Comment by Hydrox6** [comment]
You can already easily look up an NPC's location by using the Search feature of the Wiki Orb (right click it). In general too, this might be better for the wiki to do, given that they have all the needed information and this really doesn't gain anything from having direct access to client data.

**Comment by Cfourr** [comment]
You are right about it being easy to look up by using the wiki, but my thing was that I hated having to go between different windows and thought it would be a cool idea to be able to have it pop up just to right of the game like other plugins do. I mean if that's the case, then the same thing could be said about the GE plugin, which is also easy to look up with the wiki.

**Comment by Hydrox6** [comment]
Will mention that the GE plugin's sidebar also hosts the user's trades, and the search feature is simply an extra piece to recreate the ingame UI for easy use anywhere. The wiki on the other hand did not have such a price search until recently with their accurate prices initiative (https://prices.runescape.wiki)

**Comment by Cfourr** [comment]
But still you cant say that an easy to lookup side bar for monsters, that lists all kinds of other info as well as location, wouldn't be handy. A lot of other info could be added as well, the possibilities are endless, and not having to open another window to find this info would be even more helpful. Especially to those with slower computers that don't like to handle many open windows at once.
